### PR TITLE
[refman] Add link to the deprecation policy

### PR DESCRIPTION
**Pull request by proux01**
Following this Zulip discussion https://coq.zulipchat.com/#narrow/stream/237977-Coq-users/topic/Backwards.20compatibility.20w.2Er.2Et.2E.20Div0.20without.20warnings

- [ ] ~Added / updated **test-suite**.~

- [ ] ~Added **changelog**.~
- [x] Added / updated **documentation**.
  
  - [ ] ~Documented any new / changed **user messages**.~
  - [ ] ~Updated **documented syntax** by running `make doc_gram_rsts`.~

- [ ] ~Opened **overlay** pull requests.~

<!-- Pointers to relevant developer documentation:

Contributing guide: https://github.com/coq/coq/blob/master/CONTRIBUTING.md

Test-suite: https://github.com/coq/coq/blob/master/test-suite/README.md

Changelog: https://github.com/coq/coq/blob/master/doc/changelog/README.md

Building the doc: https://github.com/coq/coq/blob/master/doc/README.md
Sphinx: https://github.com/coq/coq/blob/master/doc/sphinx/README.rst
doc_gram: https://github.com/coq/coq/blob/master/doc/tools/docgram/README.md

Overlays: https://github.com/coq/coq/blob/master/dev/ci/user-overlays/README.md

### CODE REVIEW

File: doc/sphinx/using/libraries/writing.rst
Comment (jfehrle) [inline doc/sphinx/using/libraries/writing.rst:43]:
```suggestion
   Coq and its standard library follow this deprecation policy:
```

**Review by jfehrle — APPROVED** [review]
Wording suggestion.

File: doc/sphinx/biblio.bib
Comment (Zimmi48) [inline doc/sphinx/biblio.bib:652]: FWIW, the university name at the time was "Université de Paris", now officially "Université Paris Cité".

Comment (proux01) [reply doc/sphinx/biblio.bib]: Congrats for managing to graduate in the short time interval that name survived!

**Comment by jfehrle** [comment]
The job still hasn't started after 6 hours.  @proux01, perhaps you want to poke it?

@Zimmi48 If you're happy with this, I will merge once the doc builds.  (And feel free to remind me if I don't merge within 24 hours.)

**Review by Zimmi48 — APPROVED** [review]

**Comment by proux01** [comment]
@jfehrle CI finally ran

**Comment by jfehrle** [comment]
@coqbot merge now

**Comment by jfehrle** [comment]
@coqbot merge now

**Comment by proux01** [comment]
Thanks!

**Pull request merged**
